{{cookiecutter.bot_name}}/app/db/sqlalchemy.py

```diff
@@ -0,0 +1,58 @@
+"""SQLAlchemy helpers."""
+
+from typing import Any
+
+from sqlalchemy.ext.asyncio import AsyncSession, create_async_engine
+from sqlalchemy.orm import declarative_base, sessionmaker
+
+from app.settings.config import get_app_settings
+
+DATABASE_URL = get_app_settings().DATABASE_URL
+
+
+def make_url_async(url: str) -> str:
+    """Add +asyncpg to url scheme."""
+    return "postgresql+asyncpg" + url[url.find(":") :]  # noqa: WPS336
+
+
+def make_url_sync(url: str) -> str:
+    """Add +asyncpg to url scheme."""
+    return "postgresql" + url[url.find(":") :]  # noqa: WPS336
+
+
+Base = declarative_base()
+
+
+class AsyncDatabaseSession:
+    """Database session class."""
+
+    def __init__(self) -> None:
+        """Initialize."""
+        self._session = None
+        self._engine = None
+
+    def __getattr__(self, name: str) -> Any:
```

Comment:
If this class created just for setup/teardown operations, maybe separate it from class and drop class altogether? I think working with plain SQLAlchemy session type will be much cleaner.